### PR TITLE
Add card metadata to "me credit transfer" api

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

We're now storing card metadata with every transfer  

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver
- [ ] Add release notes to [unreleased changelog](https://github.com/teamsempo/SempoBlockchain/blob/master/CHANGELOG.md#unreleased)

### CODE REVIEW

File: app/server/models/transfer_card_usage.py [hunk 1]
```diff
@@ -21,3 +21,11 @@ class TransferCardUsage(ModelBase):
     # Credit transfer
     credit_transfer_id       = db.Column(db.Integer, db.ForeignKey("credit_transfer.id"), index=True)
     credit_transfer          = db.relationship('CreditTransfer', foreign_keys=[credit_transfer_id], backref=backref('transfer_card_usage', uselist=False), lazy='select')
+
+    session_number = db.Column(db.Integer, default=-1)
+    amount_loaded = db.Column(db.Integer, default=-1)
+    amount_deducted = db.Column(db.Integer, default=-1)
+    
+    @hybrid_property
+    def balance(self):
```
Comment (michielderoos) [inline app/server/models/transfer_card_usage.py:30]: This is going to be used later to calculate the difference between the transfer account balance, and the last known card balance

**Comment by michielderoos** [comment]
[Accept the numbers on the backend, store them with transfer card usage object](https://trello.com/c/lDDcb03Y/1057-accept-the-numbers-on-the-backend-store-them-with-transfer-card-usage-object)

**Comment by michielderoos** [comment]
[Add transfer card information to transfer account API](https://trello.com/c/z0zYxPND/1062-add-transfer-card-information-to-transfer-account-api)

**Review by tristanhcole — APPROVED** [review]
Generally happy with this as it doesn't touch existing flow, just adds information

**Pull request merged**